streaming/index.js

```diff
@@ -208,6 +338,7 @@ const startServer = async () => {
    * @param {string} channel
    */
   const onRedisMessage = (message, channel) => {
+    const isPublicChannel = PUBLIC_CHANNELS.includes(channel.slice('timeline:'.length));
```

Comment:
Same comment as for `FALSE_VALUES`, `PUBLIC_CHANNELS` could be a `Set` to speed-up the check.